src/partcad/runtime_python_conda.py

```
@@ -326,29 +362,34 @@ def once_conda_locked_attempt(self):
                         )
                         stdout, stderr = p.communicate()
 
+                    # Everything reported in here is a *warning*, however bad it
+                    # reads. This is one attempt of several, and whether the
+                    # sandbox ends up usable is decided by the caller
+                    # (once_conda_holding_install_lock), which retries and then
+                    # raises. pc_logging.error() sets the global had_errors flag
+                    # that the CLI turns into a non-zero exit at the very end of
+                    # the command -- so an error logged here failed a run that
+                    # had already recovered, rendered everything and finished.
                     if not stderr is None and stderr.strip() != "":
```

Ruff (0.16.2)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)